web/src/components/camera/CameraImage.tsx

```diff
@@ -48,6 +48,8 @@ export default function CameraImage({
     }
   }, [containerWidth, containerHeight, hasLoaded]);
 
+  useEffect(() => setHasLoaded(false), [camera]);
```

Comment:
Is there anything that guarantees that the `<img onLoad` wont be fired before this which sets loaded to true?

Reply:
`camera` doesn't change from its initial value in every case where we use this component except in the debug view and motion tuner, so yes, there is the possibility of a race but I think it's probably very low.